.travis.yml

```diff
@@ -13,8 +14,57 @@ env:
   matrix:
   - TEST_SUITE=vmdb PARALLEL=true
   - TEST_SUITE=brakeman
-matrix:
+matxix:
```

Comment:
typo?

Reply:
I have rebased the code and resolved the typo issue.